styleguide/source/_patterns/01-molecules/06-components/11-hub-card/hub-card-portrait.twig

```diff
@@ -0,0 +1,8 @@
+{#
+ # @file hub-card-portrait.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<a class="hub-card hub-card-portrait hub-card-image-123" href="#">
```

Comment:
yah it basically has to be a background image unless we dont' care about extra white space. I also think for accessibility purposes it is *better* for this to be a background image as it is more of a decorative element and the real "meat" of the content is the button text. As that tells the user where the link goes to. Alt text would just get in the way of that info.